anonymous_tokens/cpp/privacy_pass/demo/rsa_bssa_public_metadata_privacy_pass_server_demo.cc

```diff
@@ -42,6 +42,31 @@ absl::Status RunDemo() {
     return rsa_private_key.status();
   }
 
+  // Get base64url encoded public key for client to use.
+  absl::StatusOr<bssl::UniquePtr<RSA>> rsa_public_key =
+      anonymous_tokens::CreatePublicKeyRSA(
+          test_rsa_public_key.n, test_rsa_public_key.e);
+  if (!rsa_public_key.ok()) {
+    return rsa_public_key.status();
+  }
+
+  // Compute RSA BSSA Public Key Token ID.
```

Comment:
Remove comment "// Compute RSA BSSA Public Key Token ID" as that does not seem to be happening.

Reply:
I've updated the comment to be
```
// Compute RSA BSSA Public Key in DER encoded format.
```